sdk/src/contexts/services.rs

```diff
@@ -1,136 +1,438 @@
-    /// instance.
-    fn current_service_operators(
+    async fn current_service_operators(
         &self,
-        client: &subxt::OnlineClient<Self::Config>,
-    ) -> impl Future<Output = color_eyre::Result<Vec<(subxt::utils::AccountId32, Percent)>, subxt::Error>>;
+        client: &crate::ext::subxt::OnlineClient<Self::Config>,
+    ) -> Result<
+        Vec<(
+            crate::ext::subxt::utils::AccountId32,
+            crate::ext::tangle_subxt::tangle_testnet_runtime::api::runtime_types::sp_arithmetic::per_things::Percent,
+        )>,
+        crate::ext::subxt::Error
+    > {
+        use crate::ext::subxt;
+        use crate::ext::tangle_subxt::tangle_testnet_runtime::api;
+
+        let service_id = self.service_id()?;
+        let service_instance = api::storage().services().instances(service_id);
+        let storage = client.storage().at_latest().await?;
+        let result = storage.fetch(&service_instance).await?;
+        match result {
+            Some(instance) => {
+                let mut ret = instance.operators.0;
+                ret.sort_by(|a, b| a.0.cmp(&b.0));
+                Ok(ret)
+            },
+            None => Err(subxt::Error::Other(format!(
+                "Service instance {service_id} is not created, yet"
+            ))),
+        }
+    }
 
-    #[allow(clippy::type_complexity)]
-    /// Get metadata for a list of operators from the context.
-    fn operators_metadata(
+    async fn operators_metadata(
         &self,
-        client: &subxt::OnlineClient<Self::Config>,
-        operators: Vec<subxt::utils::AccountId32>,
-    ) -> impl Future<
-        Output = color_eyre::Result<
-            Vec<(
-                subxt::utils::AccountId32,
-                OperatorMetadata<
-                    subxt::utils::AccountId32,
-                    Balance,
-                    AssetId,
-                    MaxDelegations,
-                    MaxOperatorBlueprints,
-                >,
-            )>,
-            subxt::Error,
-        >,
-    >;
+        client: &crate::ext::subxt::OnlineClient<Self::Config>,
+        operators: Vec<crate::ext::subxt::utils::AccountId32>,
+    ) -> Result<
+        Vec<(
+            crate::ext::subxt::utils::AccountId32,
+            crate::tangle_subxt::tangle_testnet_runtime::api::runtime_types::pallet_multi_asset_delegation::types::operator::OperatorMetadata<
+                crate::ext::subxt::utils::AccountId32,
+                crate::ext::tangle_subxt::tangle_testnet_runtime::api::assets::events::burned::Balance,
+                crate::ext::tangle_subxt::tangle_testnet_runtime::api::assets::events::accounts_destroyed::AssetId,
+                crate::ext::tangle_subxt::tangle_testnet_runtime::api::runtime_types::tangle_testnet_runtime::MaxDelegations,
+                crate::ext::tangle_subxt::tangle_testnet_runtime::api::runtime_types::tangle_testnet_runtime::MaxOperatorBlueprints,
```

Comment:
```suggestion
            AccountId32,
            OperatorMetadata<
                AccountId32,
                Balance,
                AssetId,
                MaxDelegations,
                MaxOperatorBlueprints,
```
All these long type paths can be removed since it's no longer in a macro (this is true for all methods)